exo-create/features/missing-argument.feature

```diff
@@ -1,12 +1,12 @@
 Feature: Missing entity
 
   As a user unsure which entities I can create
-  I want to be able to start the exo-create command without arguments and see the possible entities
+  I want to be able to start the exo-create command without arguments
   So that I can call this command correctly and build up muscle memory.
 
   - calling "exo-create" without an argument displays the list of entities that can be created
 
 
   Scenario: the user calls "exo create" without arguments
     When running "exo-create" in the terminal
-    Then it prints "Error: missing entity for 'create' command" in the terminal
+    Then it prints "Error: missing argument for 'create' command" in the terminal
```

Comment:
Since `exo create application` only creates an application now, I wonder if we can shorten it to just `exo create` now? We no longer have to distinguish between `exo create service` and `exo create application` so the extra argument seems unnecessary here.

@kevgo thoughts?